components/vector-kubearchive-log-collector/staging/stone-stg-rh01/loki-secret.yaml

```
@@ -0,0 +1,24 @@
+---
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: loki-secret
+  annotations:
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+    argocd.argoproj.io/sync-wave: "-1"
+spec:
+  dataFrom:
+    - extract:
+        key: integrations-output/external-resources/appsres09ue1/stonesoup-infra-stage/kube-archive-staging-rds
+  refreshInterval: 1h
+  secretStoreRef:
+    kind: ClusterSecretStore
+    name: appsre-vault
+  target:
+    creationPolicy: Owner
+    deletionPolicy: Delete
+    name: kubearchive-loki-credentials
+    template:
+      data:
+        LOKI_URL: '{{ index . "loki.host" }}'
+        LOKI_PASSWORD: '{{ index . "loki.password" }}'
```

Shouldn't this be USERNAME and PASSWORD according to what we create on `loki-basic-auth` Secret in the install.sh script of the loki logging integration? See [here](https://github.com/kubearchive/kubearchive/blob/main/integrations/logging/loki/install.sh#L89-L89)

```suggestion
        USERNAME: '{{ .username }}'
        PASSWORD: '{{ .password }}'

```